aiida_quantumespresso/workflows/pw/relax.py

```diff
@@ -184,7 +184,7 @@ def results(self):
         # Get the latest workchain, which is either the workchain_scf if it ran or otherwise the last regular workchain
         try:
             workchain = self.ctx.workchain_scf
-            structure = workchain.inputs.pw.structure
+            structure = workchain.inputs.pw__structure
```

Comment:
Inputs from nested workchain are access with double underscore `__` now? 
I can not find information in documentation. 
This will be the new way to access the inputs from nested workchain?

Reply:
Well, the link label for nested namespaces is generated by replacing each nesting with double underscore. Since a double underscore is valid in a python identifier, you can use it like this, yes

Reply:
@sphuber, can you confirm that this is the *only* way to access a nested namespace now? It seems to be the case (at least for process inputs).

Reply:
through the `inputs` property, yes, that is the only way. You could call `self.node.get_incoming().nested()` which will return a nested dictionary with the reconstructed original nesting